### PR TITLE
Checking if circle ci breaks if password is an empty string

### PR DESCRIPTION
**Pull request by farhanali3193**

### CODE REVIEW

**Comment by farhanali3193** [comment]
@jabrake @tchitrakorn : DO NOT MERGE YET. Working on deploying db to ec2 and passing correct env_vars.

**Comment by farhanali3193** [comment]
@jabrake @tchitrakorn : This can be merged now. Successfully deployed db on ec2 using docker.

File: pg/index.js [hunk 1]
```diff
@@ -2,18 +2,32 @@ const { Client } = require('pg');
...
 } else if (process.env.NODE_ENV === 'test') {
   database = 'qna-test';
   user = 'postgres';
+  password = process.env.PG_DB_PASS_TEST;
+  host = 'localhost';
+} else if (process.env.NODE_ENV === 'production') {
+  database = 'qna';
+  user = 'postgres';
+  password = process.env.PG_DB_PASS_PROD;
+  host = '44.197.129.107';
```
Comment (jabrake) [inline pg/index.js:22]: Is this the IP address of the instance you deployed your DB to?

**Review by jabrake — APPROVED** [review]
Looks good!

File: pg/index.js (hunk 1 above)
Comment (farhanali3193) [reply pg/index.js:22]: @jabrake correct.

**Pull request merged**